### PR TITLE
Add ocp 4.7 ppc64le job to CI

### PR DESCRIPTION
**Pull request by Basavaraju-G**
Add OCP 4.7 ppc64le job to CI

### CODE REVIEW

**Comment by Basavaraju-G** [comment]
tests are failed with pull rate limit on ci nodes.
```
oc image append --insecure --from docker.io/library/busybox:latest --to image-registry.openshift-image-registry.svc:5000/e2e-test-image-append-qq8sc/test:busybox1 --image '{"Cmd":["/bin/sleep"]}'
error: unable to read image docker.io/library/busybox:latest: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
```

File: core-services/template-deprecation/_allowlist.yaml [hunk 1]
```diff
@@ -3025,6 +3025,10 @@ templates:
     template_name: prow-job-cluster-launch-installer-remote-libvirt-e2e
     unknown_blocker:
       description: unknown
+      jobs:
+        release-openshift-origin-installer-e2e-remote-libvirt-ppc64le-4.7:
+          generated: false
+          kind: release
```
Comment (petr-muller) [inline core-services/template-deprecation/_allowlist.yaml:3031]: Please add this to the list of jobs blocked by DPTP-1735 above

Comment (Basavaraju-G) [reply core-services/template-deprecation/_allowlist.yaml]: Done

**Comment by Basavaraju-G** [comment]
@hamzy @vijay-vin above tests failed wtih docker.io rate limit. this is fixed in this PR. https://github.com/openshift/release/pull/13766 im waiting above PR get merged.

**Review by petr-muller — APPROVED** [review]

**Pull request merged**